README.md

```diff
@@ -5,6 +5,38 @@ fedora-packages allows to search for packages in Fedora.
 ## Documentation
 Would be nice to have a bit more documentation.
 
+### Hacking with docker-compose
+We have a docker-compose setup for hacking on the fedora-packages app.
+This setup matches the production deployment of fedora-packages (ie
+apache HTTP server with mod_wsgi).
+
+To bring the environment up you should first install the following.
+
+	$ sudo dnf docker-compose
```

Comment:
dnf install?

Reply:
Also, does the docker service need to run?

Reply:
Good catch, I have updated the instruction and added couple lines about the docker service.